.tav.yml

```diff
@@ -1,6 +1,12 @@
 # This file is used by tool TAV - Test All Versions.
 # See: https://github.com/watson/test-all-versions
 
+'@apollo/server':
+  versions: ^4.0.0
```

Comment:
`@apollo/server@4.0.0` *did* support node v12 as a minimum version. However, at this point given that Node v12 is EOL, I don't think we need to add testing with node v12 unless someone comes asking for it.